cpp/ps0001/ps0001_sol.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
+int main()
+{
+    // prompting user for 3 numbers to add 1.5/2
+    std::cout<<"Hello! please enter the numbers\n";
+    unsigned short a,b,c,sum;
+    std::cin>>a>>b>>c;
+    
+    // outputting the result of the summation process 3/3
+    sum=(a+b+c);
+    std::cout<<"the sum is: "<<sum<<"\n";
+
+    return 0;
+}
+// good work 👍 7.5/10
```

Comment:
- [ ] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code